default/scripting/ship_parts/Hangars.focs.txt

```diff
@@ -1,9 +1,9 @@
 Part
-    name = "FT_HANGAR_0"
+    name = "FT_HANGAR_1"
     description = "FT_HANGAR_1_DESC"
     class = FighterHangar
     capacity = 2
-    damage = 0
```

Comment:
Leave the 0 damage fighters in. They can be relatively cheap, but still potentially useful as decoys in battle that don't increase support costs, and also can be added to monsters or other non-player-designed ships.